### PR TITLE
add location search to location page.

### PR DESCRIPTION
**Pull request by vlandham**
For #131 

Here is what it looks like:

screen shot 2016-10-14 at 10 00 33 am

screen shot 2016-10-14 at 10 00 39 am

### CODE REVIEW

**Comment by vlandham** [comment]
ping @pbeshai

File: src/containers/LocationPage/LocationPage.jsx [hunk 1]
```diff
@@ -641,14 +643,33 @@ class LocationPage extends PureComponent {
     );
   }
 
+  renderLocationSearch() {
+    return (
+      <LocationSearch />
```
Comment (pbeshai) [inline src/containers/LocationPage/LocationPage.jsx]: The problem with using the default callback is that you will lose your filter settings. Maybe you should add a custom handler that does queryRebuild to maintain them?

**Comment by vlandham** [comment]
this follows the pattern of how breadcrumbs works. 
that work for you @pbeshai ? see any caveats?

**Comment by pbeshai** [comment]
I think it's fine. There's a question of whether query should be passed in as a parameter or if we should provide a handler that encodes it, but I think it's fine.

**Comment by pbeshai** [comment]
+1

**Pull request merged**